### PR TITLE
Iterate over TC Owners when tagging indicator

### PR DESCRIPTION
**Pull request by secdevopsai**
tc-tag-indicator This function would fail sometimes because it does not go through each owner in TC

### CODE REVIEW

**Comment by asafshen** [comment]
@secdevopsai looks good

I'm not familiar with the threat connect python implementation  @meirwah can you please review also?

File: Integrations/integration-ThreatConnect.yml [hunk 1]
```diff
@@ -297,6 +298,12 @@ script:
         indicators = tc.indicators()
         filter1 = indicators.add_filter()
         filter1.add_indicator(demisto.args()['indicator'])
+        owners_arr = []
```
Comment (meirwah) [inline Integrations/integration-ThreatConnect.yml]: You should provide a way to override this with argument:
```
owners = ''
        if demisto.get(demisto.args(), 'owners'):
            owners = demisto.args()['owners']
        if len(owners) > 0:

```

Comment (secdevopsai) [reply Integrations/integration-ThreatConnect.yml]: @meirwah - added

**Comment by Itay4** [comment]
@meirwah Can you please review ?

**Pull request merged**